Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseAutonomous.java

```diff
@@ -76,11 +110,11 @@ public void turnToAngle(double targetAngle) {
 
             // robot is turning counter-clockwise
             if (angleError > 0) {
-                motorPower = Math.min(angleError / -180.0, -0.2);
+                motorPower = Math.min(angleError / -250.0, -0.05);
```

Comment:
Please use constants with informative variable names instead of magic numbers. Please fix this everywhere where you have magic numbers, not just where I have commented.

Reply:
added constants for all "magic numbers"